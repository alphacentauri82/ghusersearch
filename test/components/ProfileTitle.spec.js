import React from 'react';
import renderer from 'react-test-renderer';
import {
  StyleSheetTestUtils,
} from 'aphrodite';
import ProfileTitle from 'components/ProfileTitle';

describe('Component: ProfileTitle', () => {

  beforeAll(() => {
    StyleSheetTestUtils.suppressStyleInjection();
  });

  afterAll(() => {
    StyleSheetTestUtils.clearBufferAndResumeStyleInjection();
  });

  it('should render', () => {
    const component = renderer.create(
      <ProfileTitle
        name="Simon"
        username="simonsmith"
        userLink="http://simonsmith.io"
      />
    );
    expect(component).toMatchSnapshot();
  });

});
