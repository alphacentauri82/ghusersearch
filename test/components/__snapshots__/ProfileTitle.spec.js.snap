// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: ProfileTitle should render 1`] = `
<div
  className="u-flex u-flexWrap u-flexAlignItemsBaseline"
>
  <h1
    className="ProfileTitle_name_1oubu48"
  >
    Simon
  </h1>
  <p
    className="ProfileTitle_username_irnxte"
  >
    <a
      href="http://simonsmith.io"
    >
      @
      simonsmith
    </a>
  </p>
</div>
`;
